packages/shared/src/tableDefinitions/tracesTable.ts

```diff
@@ -138,17 +138,44 @@ export const tracesTableCols: ColumnDefinition[] = [
   },
 ];
 
-export const evalTableCols: ColumnDefinition[] = tracesOnlyCols;
+// Used only for dataset evaluator, not on dataset table
+export const datasetOnlyCols: ColumnDefinition[] = [
```

Comment:
in which UI elements is this added? This is potentially breaking with V3 in case we require a join observations x dataset

Reply:
Only used for the dataset evaluator form, no tables as per comment above. No join required :)